persistence-modules/hibernate-annotations-2/pom.xml

```diff
@@ -92,15 +57,24 @@
             <artifactId>lombok</artifactId>
             <version>${lombok.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <version>${spring-boot.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-core</artifactId>
+            <version>1.5.8</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
extract property + any idea why this isn't already pulled in by the parent hierarchy? I thought one of the parent modules had a default logback configuration etc defined

Reply:
I've extracted the property. But idk about the default config, I added this to resolve context startup exception when running tests.